### PR TITLE
feature file for story EC-46

### PR DESCRIPTION
**Pull request by shyamrgangane**
Hi Team,

Please review the feature file for story EC-46 and le me know the changes if any.

Thanks,
Person Name

### CODE REVIEW

File: src/test/resources/features/prodDesc.feature [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+  Scenario: Display Prod-s desc page
+    Given I want to write a step with precondition
+    And some other precondition
+    When I complete action
+    And some other action
+    And yet another action
+    Then I validate the outcomes
+    And check more outcomes
+    
+      @tag3e
```
Comment (sarangholey) [inline src/test/resources/features/prodDesc.feature:43]: Remove the char "e"

File: src/test/resources/features/prodDesc.feature [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    Given I want to write a step with precondition
+    And some other precondition
+    When I complete action
+    And some other action
+    And yet another action
+    Then I validate the outcomes
+    And check more outcomes
+    
+    @tag2
+  Scenario: Display Prod-s desc page
```
Comment (sarangholey) [inline src/test/resources/features/prodDesc.feature:34]: Please check the Scenario line its having "-s" remove it

**Review by sarangholey — COMMENTED** [review]
Please make the associated changes to the feature file.

File: src/test/resources/features/prodDesc.feature (hunk 1 above)
Comment (shyamrgangane) [reply src/test/resources/features/prodDesc.feature]: Made the changes

File: src/test/resources/features/prodDesc.feature (hunk 2 above)
Comment (shyamrgangane) [reply src/test/resources/features/prodDesc.feature]: Made the changes

**Comment by sarangholey** [comment]
Alright now you can merge the branch to Master

**Pull request merged**